scripts/actions/fetch-quickstarts.js

```diff
@@ -13,18 +13,18 @@ const fs = require('fs');
 const fetch = require('node-fetch');
 const get = require('lodash.get');
 
-const PACKS_FILE_PATH = './src/data/observability-packs.json';
+const PACKS_FILE_PATH = './src/data/quickstarts.json';
 const NR_API_URL = process.env.NR_API_URL;
 const NR_API_TOKEN = process.env.NR_API_TOKEN;
 
 const packQuery = `# gql 
   {
     docs {
       openInstallation {
-        observabilityPackSearch {
+        quickstartSearch {
           count
           results {
-            observabilityPacks {
+            quickstarts {
```

Comment:
These don't look to have been changed in nerdgraph yet

Reply:
[it do be like dat tho](https://staging-api.newrelic.com/graphiql?#query=%7B%0A%20%20docs%20%7B%0A%20%20%20%20openInstallation%20%7B%0A%20%20%20%20%20%20quickstartSearch%20%7B%0A%20%20%20%20%20%20%20%20results%20%7B%0A%20%20%20%20%20%20%20%20%20%20quickstarts%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)

Reply:
:nod: